project/MiMa.scala

```diff
@@ -662,7 +662,10 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.mapError"),
 
         // #21894 Programmatic configuration of the ActorSystem
-        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this")
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this"),
+
+        // #22035 Make it possible to use anything as the key in a map
+        FilterAnyProblemStartingWith("akka.cluster.ddata")
```

Comment:
can we be somewhat more specfic? e.g. filter any problems for the classes that you changed?
add it to a new "2.4.16" section